### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by visbook** [k-ivan/visual-bookmarks-chrome#74]
The latest updates were great additions k-ivan, so thanks for the time you spend on this project.

I have a request regarding Ctrl+Clicking bookmarks.

As an example, say if I had a folder called New Cars which has picture links of cars I've previously bookmarked for future consideration. Today I want to look at some of them and want 10 tabs open side by side so I can quickly switch tabs to compare the car's features.

Currently, to open the 10 tabs, I'd open V-B, find the New Car folder, click the 10 picture links **+Ctrl**, which opens 10 tabs to the right of the V-B tab.
If you click a car link without **+Ctrl** the car webpage opens and it becomes the "top" tab and switches away from the V-B tab (which some users might prefer).

Would it be possible to add an option slider where you can choose whether V-B
a) keeps the V-B tab always as the tab "in focus" after clicking links (so you don't have to remember to keep pressing the **Ctrl** key)
or 
b) switches to the new tab after clicking a link

**Comment by visbook** [comment]
Yes k-ivan, I already have it set to open in a new tab, but what happens currently is when a new tab opens, the browser immediately switches to this new tab and I have to keep clicking back to the V-B tab to return to the bookmarks screen.
As I said previously, I like to open 15+ bookmarked pages all at the same time, so I want to stay on the V-B screen, click on all the bookmarks one after another without having to repeatedly clicking back to the V-B screen.

**Comment by k-ivan** [comment]
Sorry, I was inattentive. 
To do this, I will have to work programmatically with tabs and specify a handler for all links(this is an extra code that can be avoided). I think it's unnecessary, because you can use the middle mouse button (wheel) to open a new tab without switching focus to it (this is very convenient).
